src/main/java/com/manardenza/console/ContentsUserMenu.java

```diff
@@ -0,0 +1,261 @@
+                    findRoomMenu();
+                    break;
+                case (4):
+                    VisualUserMenu.outputSplitLine();
+                    getAllUserReservationsMenu();
+                    break;
+                case (5):
+                    VisualUserMenu.outputSplitLine();
+                    userController.logoutUser();
+                    return;
+                case (6):
+                    VisualUserMenu.outputSplitLine();
+                    System.out.println("\tThank you for using our service.");
+                    System.exit(0);
+                    break;
+                default:
+                    VisualUserMenu.outputSplitLine();
+                    System.out.println(ERROR_INCORRECT_MENU_ITEM_SELECTED);
+
+            }
+        }
+    }
+
+    private List<Hotel> findHotelByNameMenu() {
+        while (true) {
+            String hotelName = visualUserMenu.getValidInputFromUser("Enter hotel name (enter \"0\" to return back to "
+                + "menu): ", InputType.STRING);
+            if (hotelName.equals("0")) {
+                return null;
+            }
+            VisualUserMenu.outputSplitLine();
+            List<Hotel> hotelList = hotelController.findHotelByName(hotelName);
+            if (hotelList.isEmpty()) {
+                System.out.println("Hotel " + hotelName + " not found.");
+            } else {
+                System.out.println("The list of found hotels by name: " + hotelName);
+                VisualUserMenu.printListInConsole(null, hotelList);
+            }
+            return hotelList;
+        }
+    }
+
+    private List<Hotel> findHotelByCityMenu() {
+        while (true) {
+            String city = visualUserMenu.getValidInputFromUser("Enter city (enter \"0\" to return back to menu): ",
+                InputType
+                    .STRING);
+            if (city.equals("0")) {
+                return null;
+            }
+            VisualUserMenu.outputSplitLine();
+            List<Hotel> hotelList = hotelController.findHotelByCity(city);
+            if (hotelList.isEmpty()) {
+                System.out.println("Hotel in " + city + " not found.");
+            } else {
+                System.out.println("The list of found hotels in: " + city);
+                VisualUserMenu.printListInConsole(null, hotelList);
+            }
+            return hotelList;
+        }
+    }
+
+    private Map<String, List<Room>> findRoomMenu() {
+        String city = visualUserMenu.getValidInputFromUser("Enter city: ", InputType.STRING);
+        Integer persons = Integer.parseInt(visualUserMenu.getValidInputFromUser("Enter the number of persons: ",
+            InputType.INTEGER));
+        Integer price = Integer.parseInt(visualUserMenu.getValidInputFromUser("Enter maximum price: ", InputType
+            .INTEGER));
+        String reservedFrom;
+        String reservedTo;
+        Date arrival;
+        Date departure;
+        for ( ; ; ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Reduce the total number of break and continue statements in this loop to use at most one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS135)